apps/services/endorsements/api/infra/endorsement-system-api.ts

```diff
@@ -45,3 +45,31 @@ export const serviceSetup =
       .xroad(Base, Client, NationalRegistry, NationalRegistryB2C)
       .liveness('/liveness')
       .readiness('/liveness')
+
+const serviceName = 'endorsement-system-api'
+const serviceCleanupWorkerName = `${serviceName}-cleanup-worker`
+export const endorsementSystemCleanUpWorkerSetup = (): ServiceBuilder<
+  typeof serviceCleanupWorkerName
+> =>
+  service(serviceCleanupWorkerName)
+    .image('services-endorsements-api')
+    .namespace(serviceName)
+    .serviceAccount(serviceCleanupWorkerName)
+    .command('node')
+    .args('--no-experimental-fetch', 'main.js', '--job=cleanup')
+    .db({ name: 'services-endorsements-api' })
+    .migrations()
+    .secrets({
+      IDENTITY_SERVER_CLIENT_SECRET:
+        '/k8s/endorsement-system-api/IDS-shared-secret',
+      NATIONAL_REGISTRY_B2C_CLIENT_SECRET:
+        '/k8s/api/NATIONAL_REGISTRY_B2C_CLIENT_SECRET',
+    })
+    .xroad(Base, Client, NationalRegistry, NationalRegistryB2C)
+    .liveness('/liveness')
+    .readiness('/liveness')
+    .extraAttributes({
+      dev: { schedule: '@hourly' },
+      staging: { schedule: '@hourly' },
+      prod: { schedule: '@hourly' },
```

Comment:
This is a bit aggressive for slow cleanups. Would e.g. `@daily` be sufficient?

Reply:
no, this is a one off fixer, so we want to see bad rows found when it runs the first time, and see 0 rows found when it runs then next time after that